SeisoLLC/cookiecutter-python
# fix: http-based generation

# Contributor Comments

At some point in the past, likely in cb4d27ff07f3e80d88f6ade225fc873449d52c89, we broke http support.  This fixes that

## Manual testing

Running `for template in git+ssh://git@github.com/seisollc/cookiecutter-python.git gh:seisollc/cookiecutter-python; do pipx run --system-site-packages cookiecutter --no-input ${template} --checkout fix-http-support --verbose; rm -rf TODO; done` should succeed, which generates a project via HTTP and SSH

## Pull Request Checklist

Thank you for submitting a contribution to cookiecutter-python!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [x] If you are adding a dependency, please explain how it was chosen.
- [x] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results.
- [x] Validate that documentation is accurate and aligned to any project updates or additions.

Don't forget our more detailed contribution guidelines
[here](https://github.com/SeisoLLC/operations/blob/main/documents/software-guidelines.md#contributing-to-a-repository).
